src/module/features/customoracles.ts

```diff
@@ -3,11 +3,11 @@ import { compact } from 'lodash'
 import { getFoundryISTableByDfId, getFoundrySFTableByDfId } from '../dataforged'
 import { cachedDocumentsForPack } from './pack-cache'
 
-export interface OracleTreeNode {
+export interface IOracleTreeNode {
```

Comment:
BTW - this change is really just to disambiguate it from OracleTreeNode (the vue component as opposed to the data object). the `I` prefix is really just a hangover from me knowing some C#; it could easily be `OracleTreeNodeData` instead

Reply:
I think this notation is just fine. I did recognize it from my C++/C# days, and it's a _tiny bit_ out of place in TS, but it does help disambiguate.

Reply:
> I think this notation is just fine. I did recognize it from my C++/C# days, and it's a _tiny bit_ out of place in TS, but it does help disambiguate.

cool, yeah - in the absence of an official "best practices" beyond casing for interfaces, i've just borrowed. dataforged is riddled with them.

in the medium-to-long-term, maybe it'd make more sense to disambiguate this sort of thing via namespacing? then this might be something like `Components.OracleTreeNode.StateData`. though... hm. now that i think about it, renaming it to `OracleTreeNodeState` might be better here than a generic `I` affix anyways. i think i'll add that to this PR.

Reply:
Let's not overthink too much. I think what you've got here is just fine.